### PR TITLE
[GitHub] Fix minor grammatical things in the issue templates

### PR DESCRIPTION
**Pull request by whisperity**

### CODE REVIEW

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4690797&bid=15683311)

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-**Describe the solution you'd like**
+**Describe the solution you would like**
 A clear and concise description of what you want to happen.
 
-**Describe alternatives you've considered**
+**Describe alternatives you have considered**
 A clear and concise description of any alternative solutions or features you've considered.
 
-**Additional context**
+**Additional context:**
```
Comment (csordasmarton) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: Why the `:` is necessary here? We do not use it in the other list items.

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4690797&bid=15683311)

File: .github/ISSUE_TEMPLATE/Feature_request.md (hunk 1 above)
Comment (csordasmarton) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: You marked my comment as resolved but you didn't change anything here.

Comment (whisperity) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: > You marked my comment as resolved but you didn't change anything here.

Right... sorry. It seems I ended up not adding the change and force-pushing the same commit. It should be good now.

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4690797&bid=15683311)

**Pull request merged**